tests/test_common/test_model/test_abc/test_dto_model.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+import unittest
+
+from minos.common import (
+    DtoModel,
+    MinosAvroProtocol,
+)
+
+
+class TestDtoField(unittest.IsolatedAsyncioTestCase):
```

Comment:
The class name of the test case does not matches the `"Test${CLASS_NAME}"` pattern

Reply:
Right, done !!!